sdcm/mgmt.py

```diff
@@ -42,16 +43,17 @@ class HostSsl(Enum):
 
     @classmethod
     def from_str(cls, output_str):
-        try:
-            output_str = output_str.upper()
-            return getattr(cls, output_str)
-        except AttributeError:
-            raise ScyllaManagerError("Could not recognize returned host ssl status: {}".format(output_str))
+        if "SSL" in output_str:
+            return HostSsl.ON
+        return HostSsl.OFF
```

Comment:
i'm not sure I understand it.. if it has DOWN SSL, will it return HostSsl.ON?

Reply:
Yes. Because, to my understanding, SSL doesn't have to do to with whether the node is up or down

Reply:
so this is very wrong...
if the table shows ```DOWN SSL```, we must represent HostSsl.OFF.
Please, fix.